merkle-tree/concurrent/src/merkle_path.rs

```diff
@@ -0,0 +1,6 @@
+#[repr(C)]
+pub struct MerklePath<const MAX_DEPTH: usize> {
```

Comment:
MAX_DEPTH is equivalent with height merkle tree or MAX_HEIGHT?

Reply:
The whole `MerklePath` stuff is not used anymore and I'm using `MAX_HEIGHT` consistently now. Thanks for pointing out!